BUILDING.md

```diff
@@ -37,15 +33,73 @@ Building: Docker >> alpine-linux-x64 >> NODE_ENABLED
 Override build-steps ? (leave empty to run pre-configured steps): j2v8
```

Comment:
Better use `all` instead of `j2v8` because you didn't explain yet what that means.